public/prelert_swimlane_vis_controller.js

```diff
@@ -263,7 +263,14 @@ module.controller('PrelertSwimlaneVisController', function ($scope, courier) {
 
       // Create a series for each severity color band,
       // plus an 'unknown' series for scores less than the 'low' threshold.
-      const colorBands = ['#e6e6e6', '#d2e9f7', '#8bc8fb', '#ffdd00', '#ff7e00', '#fe5050'];
+      //If color pickers have been used use new color choice else go to default colors.
+      const colorBands = [scope.vis.params.unknownThresholdColor ? scope.vis.params.unknownThresholdColor :'#e6e6e6',
```

Comment:
Could you define the default colors in `prelert_swimlane_vis.js`, in the `params.defaults` block. That way the inputs are filled out with the default hexadecimal colors on opening. i.e.
```
params : {
      editor : require('plugins/prelert_swimlane_vis/prelert_swimlane_vis_params.html'),
      defaults : {
        interval : { display : 'Auto', val : 'auto' },
        lowThreshold: 0,
        warningThreshold: 3,
        minorThreshold: 25,
        majorThreshold: 50,
        criticalThreshold: 75,
        lowThresholdColor : '#d2e9f7',
        warningThresholdColor : '#8bc8fb',
        minorThresholdColor : '#ffdd00',
        majorThresholdColor : '#ff7e00',
        criticalThresholdColor : '#fe5050',
        tooltipNumberFormat: '0.0',
        showLegend: true
      },
      intervalOptions...
```